clocker.bom

```diff
@@ -1,12 +1,12 @@
 brooklyn.catalog:
   brooklyn.libraries:
     - "https://oss.sonatype.org/service/local/artifact/maven/redirect?r=snapshots&g=io.brooklyn.etcd&a=brooklyn-etcd&v=2.3.0-SNAPSHOT"
-#    TODO: Replace below URLs with link to release JARs
-    - "http://localhost:8000/io/brooklyn/clocker/common/2.0.0-SNAPSHOT/common-2.0.0-SNAPSHOT.jar"
-    - "http://localhost:8000/io/brooklyn/clocker/swarm/2.0.0-SNAPSHOT/swarm-2.0.0-SNAPSHOT.jar"
-    - "http://localhost:8000/io/brooklyn/clocker/kubernetes/2.0.0-SNAPSHOT/kubernetes-2.0.0-SNAPSHOT.jar"
+    - "https://oss.sonatype.org/service/local/artifact/maven/redirect?r=releases&g=io.brooklyn.clocker&a=common&v=2.0.0"
+    - "https://oss.sonatype.org/service/local/artifact/maven/redirect?r=releases&g=io.brooklyn.clocker&a=swarm&v=2.0.0"
+    - "https://oss.sonatype.org/service/local/artifact/maven/redirect?r=releases&g=io.brooklyn.clocker&a=kubernetes&v=2.0.0"
```

Comment:
Also, copy/paste into a shell will break, unless you remember to escape the `&` - it's just nicer to have the direct URL...